src/_pytest/stepwise.py

```diff
@@ -0,0 +1,91 @@
+from _pytest.cacheprovider import Cache
+import pytest
+
+
+def pytest_addoption(parser):
+    group = parser.getgroup('general')
+    group.addoption('--sw', '--stepwise', action='store_true', dest='stepwise',
```

Comment:
This will fail the linting check, you can either:

1. install `pre-commit` and execute it:

  ```
  $ pre-commit install
  $ pre-commit run -a
  ```

2. Execute `tox -e linting`.

I recommend 1) because from that point on it will reformat/check every commit.

Commit and push the results. 👍